client/signup/steps/site-type/style.scss

```diff
@@ -78,3 +77,52 @@
 		transform: translateX( 0 );
 	}
 }
+
+.site-type__other-option {
```

Comment:
I'd love to remove some of this CSS by using `<FormTextInputWithAction />` instead of building our own version.

Reply:
I appreciate the styling affinity suggestions. This code is temporary (will be removed shortly), so in order to support its main goal (data collection) and make use of it asap, I'll merge it as is. To reiterate: we will remove this code in less than a month.

Reply:
Well, while I'm fine to trust your judgement, I disagree. 

Just to explain: my problem is that these duplications remain forever - they don't seem to get addressed, and then as we move to other projects, we forget about them. FWIW that was the case for the user type step, but it still stays duplicated to this day. I just disagree we should keep setting bad precedents for shipping unfinished code and leaving it as-is. Ideally, we should reuse the rich component base we already have, and not introduce obsolete, duplicate components and/or styles in the first place, which I'm trying to do here.

Maybe we should ask for a second opinion - someone from @Automattic/luna or @Automattic/team-calypso - I'm fine to agree with the opinion of the majority.

Reply:
We record issues to be address. Recent events have been pretty impacting re the priorities.

> I just disagree we should keep setting bad precedents for shipping unfinished code and leaving it as-is.

Again, I do not consider it fair to make generalized statements. It seems like you are referring to a larger problem and other situations. 
I disagree with discarding deadlines in the name of perfection—we need to test our assumptions efficiently.

Reply:
As stated in my [latest review](https://github.com/Automattic/wp-calypso/pull/31524#pullrequestreview-223300577) in which I approved this PR, I agree to disagree on this point.